### DISCUSSION TITLE
Do we need an separate repository for planning?

### DISCUSSION THREAD

**Discussion by FaGru3n** [eclipse-tractusx/eclipse-tractusx.github.io#224] [Ideas]
Hi all,

do we need an separate repository for planning?

#### What you wand to add?
- generate an separate repository for QG and Release things name like "sig-deploy" or "sig-planning" or something like that
#### Why?
- have separate issues options for labeling, QG's, PI planning etc.

**Comment by FaGru3n** [comment]
@danielmiehle @Siegfriedk @giterrific what are your opinions?

FYI: @kelaja and @LuisRickert

**Reply by FaGru3n** [reply]
I would like to create a new repository for all community related stuff.

Do you are ok with repository-name like `community` found a good example from https://github.com/kubernetes/community that represents all overarching members for the Org Kubernetes and could be adapted to Eclipse Tractus-X with all Planning things, Team representation, release infrormation and so on...

**Comment by Siegfriedk** [comment]
@FaGru3n as a central entry point to community? With referneces to the sigs? Do you really know what i think about it? I love it :D <3

**Reply by FaGru3n** [reply]
jep this is my approch because in this repo we could also use issues maybe for the community and also planning our stuff there, also a good starting point to intruduce new ones into the community, but first main reason would be organize our product release with different links to product-repo's

**Reply by FaGru3n** [reply]
raised a GitLab Issue https://gitlab.eclipse.org/eclipsefdn/helpdesk/-/issues/3196 and need your votes for the Eclipse Foundation

**Comment by danielmiehle** [comment]
I like the idea. Please remember that we also started a high-level community page in Docusaurus. Let's bring the topics together.
https://eclipse-tractusx.github.io/community

**Reply by FaGru3n** [reply]
sure i don´t want to separate it... just have a repo based point how we can orgnize together. The Docusaurus Page is the visual view from it 😉

**Comment by FaGru3n** [comment]
For now the repository within Eclipse Tractus X is now available https://github.com/eclipse-tractusx/community
